pulp_ansible/app/galaxy/v1/constants.py

```diff
@@ -0,0 +1,2 @@
+LEGACY_DISTRIBUTION_PATH = "legacy"
+LEGACY_REPOSITORY_NAME = "legacy"
```

Comment:
I imagine these will be configurable as settings and these would be the defaults. Is that your plan also?

To do that define them [here](https://github.com/pulp/pulp_ansible/blob/main/pulp_ansible/app/settings.py) and then document them [here](https://github.com/pulp/pulp_ansible/blob/main/docs/settings.rst).

Reply:
Makes sense. I'll give it a shot.